.github/workflows/pytest.yml

```
@@ -0,0 +1,38 @@
+name: httomolib CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
```

If you want cirun to spinup runners for this job, you need to change this line to match the labels defined in your `.cirun.yml`:

```suggestion
    runs-on: cirun-gpu-runner
```

---

Ahh, thank you, Amit. Just playing with this for the first time!